### PR TITLE
Add initial content for Android Play Store release guide

### PR DESCRIPTION
**Pull request by mfurtak**

### CODE REVIEW

File: docs/getting-started/android/release-deployment.md [hunk 1]
```diff
@@ -1,10 +1,66 @@
-# Deploy to Google Play using fastlane
+# Deploy to Google Play using _fastlane_
```
Comment (KrauseFx) [inline docs/getting-started/android/release-deployment.md]: We don't have this on the other guides yet using _

File: docs/getting-started/android/release-deployment.md [hunk 2]
```diff
@@ -1,10 +1,66 @@
...
-    - Mention how to deploy to the beta track
-- Best practises around the deployment
-    - Order of commands
+_fastlane_ takes care of building your app by delegating to your existing Gradle build. Just add the following to your `Fastfile`:
+
+```ruby
+lane :playstore
+  gradle(
+    task: 'assemble',
+    flavor: 'Paid',
```
Comment (asfalcone) [inline docs/getting-started/android/release-deployment.md]: Not sure if `flavor: 'Paid'` is helpful here. It won't work for most people if they copy and paste this, where assemble, Release would

File: docs/getting-started/android/release-deployment.md [hunk 3]
```diff
@@ -1,10 +1,66 @@
...
+    - There is at most one of each type
+- Screenshots from `fastlane/metadata/android` if you previously ran _screengrab_
+
+If you would like to capture and upload screenshots automatically as part of your deployment process, check out the [fastlane screenshots for Android](screenshots.md) guide to get started!
+
+To gradually roll out a new build you can use:
+
+```ruby
+lane :playstore
+  # ...
```
Comment (KrauseFx) [inline docs/getting-started/android/release-deployment.md]: In the iOS guides I was just using `...` without the `#`, but this makes more sense, we should update the other guides too

**Comment by KrauseFx** [comment]
Looks good, I added 2 comments, which require action from my side to update the rest of the docs

**Comment by KrauseFx** [comment]
CI: 

> [18:32:29]: Use _supply_ instead of `supply` to mention a tool in the docs in 'docs/getting-started/android/release-deployment.md'

**Comment by asfalcone** [comment]
Just the one comment, otherwise looks great 👍

**Comment by KrauseFx** [comment]
https://github.com/fastlane/docs/pull/90

**Comment by KrauseFx** [comment]
And https://github.com/fastlane/docs/pull/91

**Comment by mfurtak** [comment]
Hm, yeah, can't refer to action names in code like `supply`, unfortunately. :(

File: docs/getting-started/android/release-deployment.md (hunk 2 above)
Comment (mfurtak) [inline docs/getting-started/android/release-deployment.md]: OK, removed

**Comment by mfurtak** [comment]
Good to go?

**Comment by asfalcone** [comment]
LGTM

**Pull request merged**